content/api/api-metrics.md

```diff
@@ -7,112 +7,685 @@ author: Lawrence Lane
+
+
+### Response Body
+
+The following response body
+```
+{
+  "metricFqn": "aws.applicationelb.httpcode_elb_4xx_count",
+  "timeSpan": {
+    "startTime": "2020-01-30T06:00:00Z",
+    "endTime": "2020-01-31T06:00:00Z",
+    "duration": {
+      "seconds": 86400,
+      "nano": 0,
+      "zero": false,
+      "negative": false,
+      "units": [
+        "SECONDS",
+        "NANOS"
+      ]
+    }
+  },
+  "metrics": [
+    {
+      "elementId": "6bdf4fd1-7134-3959-9c36-17eb5f628e46",
+      "metricId": "71d31aac-c1ad-30e1-bd1a-8f3d4ee816fd",
+      "statistics": {
+        "avg": 1,
+        "min": 1,
+        "median": 1,
+        "max": 1,
+        "percentile75": 1,
+        "percentile95": 1,
+        "count": 43,
+        "sum": 43,
+        "stddev": 0,
+        "percentile5": 1,
+        "percentile25": 1
+      }
+    },
+    {
+      "elementId": "861687fd-a348-3f05-92e1-a681d7d3d9e4",
+      "metricId": "5ed2adcc-0a1e-3919-91fc-38b159f9d4c1",
+      "statistics": {
+        "avg": 1,
+        "min": 1,
+        "median": 1,
+        "max": 1,
+        "percentile75": 1,
+        "percentile95": 1,
+        "count": 41,
+        "sum": 41,
+        "stddev": 0,
+        "percentile5": 1,
+        "percentile25": 1
+      }
+    }
+  ]
+}
+
+```
+
+{{% /expand %}}
+
+---
+
+## POST to /metrics/statistics
+
+{{< button href="https://app.metricly.com/swagger-ui.html#!/metrics/getMetricStatisticsPageUsingPOST" theme="success" >}} POST {{< /button >}} This endpoint is deprecated.   
+
+{{% expand "View Method Details." %}}
+
+### Parameters
+
+| Parameter | Parameter Type | Data Type | Description |
+|------------|----------------|---------------|-----------------------------------------------------------------------------------------|
+| fqn | query | string | The fully qualified name for a metric. |
+| duration | query | string | Gives CloudWisdom an ISO 8601-formatted duration time frame to retrieve data. The duration ends at the current time and begins anytime in the past two weeks. The duration parameter will take precedence over startTime and endTime if all attributes are included in your request. |
+| startTime | query | date-time | Start time for the query. YYYY-MM-DDT00:001Z format (must include time zone). |
+| endTime | query | date-time | End time for the query. YYYY-MM-DDT00:001Z format (must include time zone). |
+| rollup | query | Array[string] | Determines frequency of metric sampling. Options are: ZERO, PT5M, PT1H, PT24H. |
+| showValues | query | boolean | Provides the values for the metric statistics (if set to true) or only displays zeroes for the statistics (if set to false). |
+|elementIds   | body   | Array[string]  | List of elementIds. |
+
+
+### Request URL
+
+```
+https://app.metricly.com/metrics/statistics?fqn=aws.applicationelb.requestcountpertarget&startTime=2020-02-04T00%3A25%3A46.872Z&endTime=2020-02-05T00%3A25%3A46.872Z&rollup=PT5M&showValues=false
```

Comment:
Use the copy-able code span here too.

URL should be https://app.metricly.com/metrics/statistics?fqn={fqn}&startTime={startTime}&endTime={endTime}&rollup={rollup}&showValues={showValues}